html/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="../css/style.css" />
+        <title>Campeonas'22_Pharmacy school</title>
+    </head>
+    <body>
+        <header>
+            <h1>School of Pharmacy</h1>
+            <nav>
+                <a href="#importance">Importance of pharmacy profession</a>
+                <br />
+                <a href="#field">Fields in pharmacy</a> <br />
+                <a href="#quality">Qualities of a good pharmacist</a>
+            </nav>
+        </header>
+        <p>
+            Pharmacy is a course of study which produces professionals in the
+            field of drug manufacturing, production, componding, distribution
+            and use. It is act, practice or profession of preparing drugs
+            according to Doctor's prescription. The pharmacy profession is
+            cordinated by the body
+            <a href="pcn.html" title="pcn">Pharmacists Council of Nigeria (PCN)</a>.
```

Comment:
Within the paragraph, create at least three different links to external websites that provide additional information on your chosen topic.
The links are not leading to external websites